romancal/jump/tests/test_jump_step.py

```diff
@@ -142,7 +144,7 @@ def test_one_CR(generate_wfi_reffiles, max_cores, setup_inputs):
                           deltatime=grouptime)
 
     for i in range(ngroups):
-        model1.data[i, :, :] = deltaDN * i
+        model1.data.value[i, :, :] = deltaDN * i
```

Comment:
I would have said it's more idiomatic to change deltaDN -> deltaDN * u.DN that it is to have all the data.values.